scripts/start.sh

```diff
@@ -58,6 +58,10 @@ gzip -f -9 -k /www/index.html
 # gzip metadata.json again because we changed it
 gzip -f -9 -k /www/metadata.json
 
+# Create file to include into nginx config to use
+# the pods nameserver as resolver
+echo resolver $(awk 'BEGIN{ORS=" "} $1=="nameserver" {print $2}' /etc/resolv.conf) ";" > /etc/nginx/config/resolvers.conf
```

Comment:
Can you elaborate for my puny mind what this is actually doing? Where are the parameters coming from?

Reply:
Rewritten it easier